src/main/java/com/softserveinc/ita/rozetka/page_objects/CategoryPage.java

```diff
@@ -30,4 +32,7 @@ public String getPageTitle() {
         return $x("//h1[@class='portal__heading']").text();
     }
 
+    public String getPageUrlTitle() {
+        return Selenide.title();
+    }
```

Comment:
misunderstood with page title (header in the page) and page URL title.

divide these into:
page title - string title on the page
URL title - title in URL